src/actions/toggle.js

```diff
@@ -0,0 +1,3 @@
+module.exports = (state, action) => ({
+  view: action.view,
```

Comment:
Actions should now execute with action.end(newState).

The new state should be an object passed as the argument to the action.end method.

Instead of:
view: action.view,
the above should be

action.end(
   {
      view: action.view,
   }
)